index.html

```diff
@@ -1666,22 +1666,18 @@ <h5>Implementations</h5>
         <p>These messages help peers discover new pubs where they might be able to fetch new messages from. Clients can maintain a list of pubs they have seen so that they know of alternative places to fetch messages from in case a pub goes offline.</p>
 
         <!-- rooms documentation start -->
-        
+
         <h2 id="rooms">Rooms</h2>
-        
+
         <p>Rooms are an publicly accessible Scuttlebutt peer, but unlike <a href="#pubs">pubs</a> they don't host feed data from other peers. The main feature rooms provide is a way for peers to find each other and establish tunnelled connections among themselves as if they were on the same network. They also provide more moderation tools than a pub with multiple user-levels and different privacy modes.</p>
         <aside>
             <p>There are two rooms specifications. There is a legacy <a href="https://github.com/staltz/ssb-room" target="_blank">room 1.0</a> implementation but most users and room operators have moved to the new <a href="https://ssb-ngi-pointer.github.io/rooms2/" target="_blank">rooms 2.0</a> specification, which is documented here.</p>
         </aside>
         <p>Rooms differ from pubs because even though they're a Scuttlebutt peer, they are not meant to aid replication by <i>diminishing the number of hops between peers</i>. Instead, they provide a simple way for peers to find and connect to each other even though these peers might not have publicly accessible connections, and thus no <i>follow</i> messages are used in the rooms workflow.</p>
-        
+
         <h3 id="rooms-tunnel">Tunnel</h3>
 
-        <p>A tunnel is an indirect connection between two Scuttlebutt peers that is facilitated by another peer with a stable and publicly available Internet connection. This facilitator peer wraps the indirect connection between the other peers thus providing a tunnel for them to meet with one another as if they were on the same network.</p>
-        <aside>
-            <p>For a deeper understanding of tunneling, refer to <a href="https://github.com/ssbc/ssb-tunnel" target="_blank">ssb-tunnel documentation.</a>
-        </aside>
-        <p> 
+        <p>A tunnel is an indirect connection between two Scuttlebutt peers that is facilitated by another peer with a stable and publicly available Internet connection. This facilitator peer, also known as the "portal" or the room, wraps the indirect connection between the other peers thus providing a tunnel for them to meet with one another as if they were on the same network.</p>
```

Comment:
Here the term `tunnel` is used, but the previous section talks about `tunnelled connections`, if they are the same, the same term should be used.

Reply:
Seems like you understood that they are the same, so there's no problem here. The only mention of `tunnelled connections` is in the overview introduction of "Rooms" section. At that point it's not specifying anything.